EBYT.py

```diff
@@ -1,20 +1,17 @@
-
-import os
-os.environ['QT_API'] = 'PySide6' # For qasync to know which binding is being used
-os.environ['QT_LOGGING_RULES'] = 'qt.pointer.dispatch=false' # Disable pointer logging
```

Comment:
Any reason why this is removed? Without it I get errors like
qt.pointer.dispatch: skipping QEventPoint(id=1 ts=0 pos=0,0 scn=819.215,-270.472 gbl=819.215,-270.472 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-819.215,270.472 last=-819.215,270.472 Δ 819.215,-270.472) : no target window

Reply:
Environment variables should be set in the environment. We should add a script or something to set these rather than putting it in the code in the middle of the imports